### PR TITLE
docs(tutorial): french translation(_datas, pt5)

### PR DESCRIPTION
**Pull request by cbalit**
Thank you for submitting this PR...

**Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit-message-format
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**Please provide the issue number?**

issue #38

**Did you finish translating the whole issue?**
- [ ] no (pr_state: WIP)
- [x] yes (pr_state: PLEASE_REVIEW)

**What kind of change does this PR introduce?** (check one with "x")
- [x] Translation
- [ ] Other... Please describe:

**Other information**:

### CODE REVIEW

File: public/docs/ts/latest/tutorial/toh-pt5.jade [hunk 1]
```diff
@@ -2,37 +2,72 @@ include ../_util-fns
...
     pop out the preview window by clicking the blue 'X' button in the upper right corner:
 
+    Pour observer les changements d'URL dans la barre d'adresse du navigateur,
+    affichez la fenêtre de prévisualisation en cliquant sur le bouton bleu marqué d'un 'X' dans le coin en haut à droite&nbsp;:
+
 .l-main-section
 :marked
   ## Where We Left Off
+
+  ## Ou nous étions-nous arrêtés ?
```
Comment (florianorpeliere) [inline public/docs/ts/latest/tutorial/toh-pt5.jade]: Où

File: public/docs/ts/latest/tutorial/toh-pt5.jade [hunk 2]
```diff
@@ -2,37 +2,72 @@ include ../_util-fns
...
+
   Before we continue with our Tour of Heroes, let’s verify that we have the following structure after adding our hero service
   and hero detail component. If not, we’ll need to go back and follow the previous chapters.
 
+  Avant de continuer notre Guide des Héros, vérifions que nous avons bien la structure suivante après avoir ajouté notre service héros
+  et le composant de détail d'un héros.Si ce n'est pas le cas, vous devez revenir en arrière et suivre les étapes du chapitre précédent.
```
Comment (florianorpeliere) [inline public/docs/ts/latest/tutorial/toh-pt5.jade]: héros. Si (un petit espace ;) )

File: public/docs/ts/latest/tutorial/toh-pt5.jade [hunk 3]
```diff
@@ -54,133 +89,285 @@ figure.image-display
...
   ## Add Routing
 
+  ## Ajoutons le routage
+
   We're ready to take the next step.
   Instead of displaying heroes automatically, we'd like to show them *after* the user clicks a button.
   In other words, we'd like to navigate to the list of heroes.
 
+  Nous sommes prêts à passer à l'étape suivante.
+  Plutôt que d'afficher automatiquement la liste de héros, nous voudrions les montrer *après* que l'utilisateur ait cliqué sur un bouton.
```
Comment (florianorpeliere) [inline public/docs/ts/latest/tutorial/toh-pt5.jade]: des héros

File: public/docs/ts/latest/tutorial/toh-pt5.jade [hunk 4]
```diff
@@ -190,422 +377,822 @@ code-example(language="bash").
...
+  ### Le routage vers le détail d'un héros
+
   We'll add a route to the `HeroDetailComponent`  in the `AppComponent` where our other routes are configured.
 
+  Nous allons ajouter une route vers le `HeroDetailComponent` dans le `AppComponent` là ou les autres sont configurées.
+
   The new route is a bit unusual in that we must tell the `HeroDetailComponent` *which hero to show*.
   We didn't have to tell the `HeroesComponent` or the `DashboardComponent` anything.
 
+  Cette nouvelle route est un peu inhabituelle dans la mesure ou nous devons indiquer au `HeroDetailComponent` *quel héros afficher*.
```
Comment (florianorpeliere) [inline public/docs/ts/latest/tutorial/toh-pt5.jade]: où

Comment (florianorpeliere) [inline public/docs/ts/latest/tutorial/toh-pt5.jade]: héro

File: public/docs/ts/latest/tutorial/toh-pt5.jade [hunk 5]
```diff
@@ -701,18 +1331,52 @@ figure.image-display
...
+
+  - Nous avons ajouté le *Composant Routeur* de Angular pour naviguer à travers les différents composants
+
   - We learned how to create router links to represent navigation menu items.
+
+  - Nous avons appris à créer des liens de routage pour représenter des entrées dans un menu de navigation.
+
   - We used router parameters to navigate to the details of user selected hero.
+
+  - Nous avons utiliser les paramètres de routes pour naviguer vers le détail d'un héros sélectionné.
```
Comment (Aku-Aku) [inline public/docs/ts/latest/tutorial/toh-pt5.jade]: Nous avons utilisé

File: public/docs/ts/latest/tutorial/toh-pt5.jade [hunk 6]
```diff
@@ -614,36 +1201,67 @@ figure.image-display
...
+
+  ### Style global de l'application
+
   When we add styles to a component, we're keeping everything a component needs
   &mdash; HTML, the CSS, the code &mdash; together in one convenient place.
   It's pretty easy to package it all up and re-use the component somewhere else.
 
+  Lorsque nous ajoutons du style à un composant, nous gardons regroupé tout ce dont le composant à besoin
+  &mdash; le HTML, la CSS, le code &mdash;  dans un endroit aproprié.
+  Il est ainsi plus facile de tout packager pour réutiliser le composant ailleur.
```
Comment (Aku-Aku) [inline public/docs/ts/latest/tutorial/toh-pt5.jade]: ailleurs

File: public/docs/ts/latest/tutorial/toh-pt5.jade [hunk 7]
```diff
@@ -54,133 +89,285 @@ figure.image-display
...
+
   * Tie the *Dashboard* into the navigation structure
 
+  * Ajouter ce *tableau de bord* à la navigation
+
 .l-sub-section
   :marked
     *Routing* is another name for *navigation*. The *router* is the mechanism for navigating from view to view.
 
+    *Le routage* est un synonyme de la *navigation*. Le *routeur* correspond au mécanisme impliqué dans la navigation entre les vues.
```
Comment (Gillespie59) [inline public/docs/ts/latest/tutorial/toh-pt5.jade]: je supprimerai le "la" après "synonyme de"

**Comment by manekinekko** [comment]
@cbalit @Gillespie59 @florianorpeliere je merge cette PR. Si vous trouvez d'autres fautes, envoyez des PR.
Merci ;)

**Pull request merged**